contrib/scripts/upgrade_test_scripts/v12/run_gaia_v11.sh

```diff
@@ -72,4 +70,4 @@ enable = true/g' $NODE_HOME/config/app.toml
 pwd
 ls $NODE_HOME
 
-$COSMOVISOR run start --home $NODE_HOME --x-crisis-skip-assert-invariants
+$COSMOVISOR run start --home $NODE_HOME --x-crisis-skip-assert-invariants > v11.out 2>&1 &
```

Comment:
Any idea why `> v11.out 2>&1 &` was needed?

Reply:
its not needed, just to keep the same as previous upgrade script.